promscale/quick-start.md

```diff
@@ -0,0 +1,64 @@
+# Promscale quick start
+Use Docker Compose to learn how Promscale is a single data store for your
+metrics and traces with full Prometheus and OpenTelemetry support, including seamless
+integration with Grafana and Jaeger.  
```

Comment:
```suggestion
This section gets you up and running quickly with Promscale. It uses the Docker Compose tool to install everything you need to explore how Promscale can be used as a single data store for your
metrics and traces with full Prometheus and OpenTelemetry support. It also includes some additional pre-configured tooling for you to get familiar with additional Promscale functionality, such as performance monitoring and visualization tools.  
```

Reply:
I would also add a para afterwards saying something like "To learn more about how Promscale can improve your observability environment, see [About Promscale]. For instructions on installing Promscale directly, see [Installing Promscale]."